code/player/Player.cs

```diff
@@ -138,9 +138,15 @@ public override void Simulate(IClient cl)
     {
         if (Input.Pressed("view"))
         {
+            Log.Info($"{ThirdPersonCamera}");
             ThirdPersonCamera = !ThirdPersonCamera;
         }
 
+        if (Input.Pressed("swap"))
```

Comment:
Consider simplifying this UX flow; instead of having a 2nd key that you hit when already in 3rd person camera, you could just hit the 3rd person camera key again and it switches to the left shoulder camera, and if you hit it again, goes back to first person.

Reply:
Oh I totally forgot this is what will appear in the actual menu. Got it.

Reply:
If they hit the third person camera key again, I think it would be pretty annoying to have to swap like that. You'd have to go through left shoulder, right, shoulder, and first person again. It won't save the state either.